dev/idm-am-common-user/idm/script/augmentSecurityContext.js

```diff
@@ -0,0 +1,13 @@
+var authzHeaderValue = httpRequest.getHeaders().getFirst(authzHeaderName);
```

Comment:
I doubt you need this script. I only added it as a means of passing extra security context details from IG to IDM